tests/core/gadget-update-pc/task.yaml

```diff
@@ -88,7 +88,7 @@ prepare: |
     {"snap-id": "$PC_SNAP_ID", "snap-revision": "$((START_REVISION+2))"}
     EOF
 
-    snap install pc_x1.snap
+    snap install --amend pc_x1.snap
```

Comment:
git commit --amend?

Reply:
Hm or did you mean snap refresh --amend?

Reply:
Ah yeah I forgot it's refresh --amend sorry

Reply:
+ snap install --amend pc_x1.snap
error: unknown flag `amend'

Reply:
I updated the pr to make refresh but the test is still failing

Reply:
yeah I need to dig into this more